projects/storefrontlib/shared/components/popover/popover.directive.ts

```diff
@@ -237,4 +303,6 @@ export class PopoverDirective implements OnInit {
     protected popoverService: PopoverService,
     protected winRef: WindowRef
   ) {}
+  @Optional() featureFlagService = inject(FeatureConfigService);
+  @Optional() selectFocusUtility = inject(SelectFocusUtility);
```

Comment:
Same comment as [here](https://github.com/SAP/spartacus/pull/18609#discussion_r1521038283). Lets also put these at the top of the class as that's where we would usually expect to see our injected services.

Reply:
Moved to top and added the optional flag

Reply:
Lets also add our TODO cleanup comments. ie. `featureFlagService` to be removed and `selectFocusUtility` to be required next major.

Reply:
done